pkg/apis/batch/validation/validation_test.go

```diff
@@ -919,7 +919,7 @@ func TestValidateJobUpdate(t *testing.T) {
 				job.Spec.ManualSelector = pointer.BoolPtr(true)
 			},
 		},
-		"immutable completion": {
+		"immutable completions for non-indexed jobs": {
```

Comment:
also have a test case for immutable indexed job when AllowElasticIndexedJobs=false

Reply:
Added